gapic/samplegen/samplegen.py

```diff
@@ -332,23 +332,30 @@ def validate_and_transform_request(self,
 
                 if attr.message:
                     base = attr.message
-
+                elif attr.enum:
+                    # A little bit hacky, but 'values' is a list, and this is the easiest
+                    # way to verify that the value is a valid enum variant.
+                    witness = next((e.name
```

Comment:
Can't you use `any`? `witness = any([e.name == val for e in attr.enum.values])` It reads a lot cleaner, IMO. And in line 348, you can use `f"'{val}'"`

Reply:
`any` returns a boolean. If we were on python 3.8 I could use `any` and assignment expressions in something like
```
if any((witness := e.name). == val for e  in attr.enum.values):
...
else:
   raise ...
```
which is only marginally clearer, IMO.